src/driver/franka_plan_runner.h

```diff
@@ -212,6 +212,40 @@ class FrankaPlanRunner {
   franka::JointPositions JointPositionCallback(
       const franka::RobotState& robot_state, franka::Duration period);
 
+  /**
+   * @brief Helper function to determine if the start of new plan is far from
+   * the current robot conf.
+   *
+   * @param params - parameters from which to get comparison thresholds
+   * @param franka_start_conf - current robot conf from which the robot will
+   * start executing plan
+   * @param start_conf_plan - start conf of the new plan
+   * @return true, if the maximum angular distance between current robot conf
+   * and start of new plan is greater than kMediumJointDistance
+   * @return false, otherwise
+   */
+  static bool IsStartFarFromCurrentJointPosition(
+      const RobotParameters& params, const Eigen::VectorXd& franka_start_conf,
+      const Eigen::VectorXd& start_conf_plan);
```

Comment:
This seems like it could be rewritten as more generic function to compare two confs to some tolerance, like `IsAtConf(conf1, conf2, tolerance)`

Reply:
This function also prints a bunch of warnings and @dmsj wanted to wrap all that into this function. We could wrap the two lines into another function, but then we'd need the distance to print warnings if needed.